<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Earls-MacBook-Pro.local" name="SecondTest" tests="2" failures="1" timestamp="30 Oct 2015 22:29:11 GMT" time="27.892" errors="0">
  <testcase name="login" time="8.644" classname="LearningTestNG.TestNGAnotation_1"/>
  <testcase name="logout" time="3.61" classname="LearningTestNG.TestNGAnotation_1">
    <failure type="java.lang.AssertionError" message="expected [Sign In] but found [SIGN IN]">
      <![CDATA[java.lang.AssertionError: expected [Sign In] but found [SIGN IN]
	at LearningTestNG.TestNGAnotation_1.logout(TestNGAnotation_1.java:113)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- logout -->
</testsuite> <!-- SecondTest -->
